app/models/v2/website.rb

```diff
@@ -0,0 +1,151 @@
+class V2::Website
+
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  field :url, type: String
+  field :first_test, type: String
+  field :cms, type: String
+
+  def self.import_urls(urls)
+    urls_id = []
+    urls.each do |url|
+      _url = self.where(url: url).first
+      if _url
+        urls_id << website_id
+      else
+        _url = self.create(url: url, site_data_info_id: nil)
```

Comment:
i think we don't need to maintain site_data_info_id anymore we can fetch it direct from last site_data_info of url and test